docs/source/gateway.md

```diff
@@ -38,39 +34,32 @@ Hyperledger Fabric currently supports client application development in three la
+- The transaction proposal is simulated on the selected endorsement peer. This simulation captures information about the accessed states, and therefore the endorsement policies to be combined (including any individual state-based policies stored on the endorsement peer's ledger).  
 - The captured policy information is assembled into a `ChaincodeInterest` protobuf structure and passed to the discovery service in order to derive an endorsement plan specific to the proposed transaction.
 - The gateway applies the endorsement plan by requesting endorsement from the organizations required to satisfy all policies in the plan. For each organization, the gateway peer requests endorsement from the (available) peer with the highest block height.
 
 The gateway is dependent on the [discovery service](discovery-overview.html) to get the connection details of both the available peers and ordering service nodes, and for calculating the combination of peers that are required to endorse the transaction proposal. The discovery service must therefore always remain enabled on peers where the gateway service is enabled.
 
-The gateway endorsement process is more restrictive for private data passed in the proposal as transient data because it often contains sensitive or personal information that must not be passed to peers of all organizations. In this case, the gateway will restrict the set of endorsing organizations to those that are members of the private data collection to be accessed (either read or write). If this restriction for transient data would not satisfy the endorsement policy, the gateway returns an error to the client rather than forwarding the private data to organizations that may not be authorized to access the private data. In these cases, client applications should be written to [explicitly define which organizations should endorse](#targeting-specific-endorsement-peers) the transaction.
+### Endorsing private data
 
-### Targeting specific endorsement peers
+The gateway endorsement process is more restrictive for private data, passed in a transaction proposal as transient data, because it often contains sensitive or personal information that must never be shared with all organizations. In this case, the gateway will restrict the set of endorsing organizations to only members of the private data collection to be accessed (either read or write). If this restriction for private data would not satisfy the endorsement policy, the gateway returns an error to the client and does not submit the transaction to any organization for endorsement. In this case, the client application must explicitly [specify the organizations](#targeting-specific-endorsement-peers) to request endorsement from.
```

Comment:
Link needs to be updated as well.

```suggestion
The gateway endorsement process is more restrictive for private data, passed in a transaction proposal as transient data, because it often contains sensitive or personal information that must never be shared with all organizations. In this case, the gateway will restrict the set of endorsing organizations to only members of the private data collection to be accessed (either read or write). If this restriction for private data would not satisfy the endorsement policy, the gateway returns an error to the client and does not submit the transaction to any organization for endorsement. In this case, the client application must explicitly [specify the organizations](#specifying-endorsement-organizations) to request endorsement from.
```

Reply:
done.